corehq/apps/export/export.py

```diff
@@ -48,19 +47,18 @@ def __enter__(self):
 
     def __exit__(self, exc_type, exc_val, exc_tb):
         self.file.close()
-        os.remove(self.path)
```

Comment:
This is the old way these were cleaned up - via `with` statement used when the generated file was read.  Removing this because `TransientTempfile` handles the cleanup now.